### PR TITLE
Save file

### PR DESCRIPTION
**Pull request by jburel**
Fix "overwrite" problem
To test the PR
test 1:
 * Open an image with mutli z and t in ImageJ
 * Click on BF-exporter
 * Select a name that already exists in chosen directory (if none, save first as ome-tiff)
 * to make sure the image is correctly updated, you should crop the original image
 * Click ok when the question to overwrite the file is asked.
 * Click ok for each of the following dialogs popping up
 * Check that the generated file is correct

test 2
* Repeat the steps from test 1 but this time select write z, t, c. 
* You will probably have to repeat twice to first generate the file with plane information.

### CODE REVIEW

File: components/bio-formats-plugins/src/loci/plugins/out/Exporter.java [hunk 1]
```diff
@@ -560,6 +559,34 @@ else if (proc instanceof ShortProcessor) {
...
+            boolean in = false;
+            for (int i = 0; i < outputFiles.length; i++) {
+                String name = new File(outputFiles[i]).getName();
+                for (int j = 0; j < files.length; j++) {
+                    if (files[j].equals(name)) {
+                        in = true;
+                        break;
+                    }
+                }
+            }
```
Comment (melissalinkert) [inline components/bio-formats-plugins/src/loci/plugins/out/Exporter.java]: Could this be simplified a bit?  Something like:

```
boolean in = false;
for (String file : outputFiles) {
  if (new File(file).exists()) {
    in = true;
    break;
  }
}
```

Comment (jburel) [inline components/bio-formats-plugins/src/loci/plugins/out/Exporter.java]: Yes, I should have done that, I was testing the file output and did not simplify.
Too quick to move to other tasks

**Comment by will-moore** [comment]
When you say "Open an image" - Using what software? / setup?

**Comment by jburel** [comment]
@will-moore: sorry in ImageJ

**Comment by bramalingam** [comment]
Everything works as expected. But a small fix might be required for the number of dialogs that pop up , when the full file name with extension is typed in while overwriting(was introduced by me previously). Fixing that in this PR itself would be great.

**Comment by jburel** [comment]
@bramalingam: saving now fixed
Also stop the saving process when the user clicks on Cancel in The "split" dialog.

**Comment by bramalingam** [comment]
Everything works as expected. Please merge.

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**